src/docs/content/observability/observability-example/_index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Observability Example"
+date: 2020-10-26 T16:04:15-06:00
+draft: false
+weight: 110
+showChildren: true
+---
+
+## Introduction
+
+In this section we will try to understand Observability on Oracle Cloud Infrastructure (OCI) by simulating a failure scenario on MuShop application.
+
+![observability-scenario](../images/observability-scenario.png)
+
+## Scenario Details
+- User logs in to MuShop and Orders items with value more than $105.
```

Comment:
Add a note that currently the Payments service is configured to decline all orders above $105

Reply:
nit: "... orders items...."

Reply:
fixed https://github.com/oracle-quickstart/oci-cloudnative/pull/215/commits/54180145ccade73eca7af95ec316bf0d65372af7